### PR TITLE
cleanups

### PR DESCRIPTION
**Pull request by SeanFeldman**

### CODE REVIEW

File: src/Transport/Config/QueueAddress.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
     using System;
     using Microsoft.Azure.Cosmos.Table;
 
-    readonly struct QueueAddress : IEquatable<QueueAddress>
```
Comment (danielmarbach) [inline src/Transport/Config/QueueAddress.cs:6]: Why is this removed?

**Review by danielmarbach — APPROVED** [review]

File: src/Transport/Config/QueueAddress.cs (hunk 1 above)
Comment (SeanFeldman) [reply src/Transport/Config/QueueAddress.cs]: R# 🙂

Comment (danielmarbach) [reply src/Transport/Config/QueueAddress.cs]: Yeah that is a false positive
